packages/pages/src/dev/dev.ts

```diff
@@ -16,13 +19,39 @@ const handler = async ({
   "prod-url": useProdURLs,
   "open-browser": openBrowser,
 }: DevArgs) => {
+  await autoYextInit();
   await createServer(!local, !!useProdURLs, scope);
   if (!openBrowser) {
     return;
   }
   await open(`http://localhost:${devServerPort}/`);
 };
 
+const autoYextInit = async () => {
+  if (!fs.existsSync(".yextrc")) return;
+  try {
+    const yextrcContents: string = fs.readFileSync(".yextrc", "utf8");
+    const parsedContents = YAML.parse(yextrcContents);
+    const businessId: string = parsedContents.businessId;
+    const universe: string = parsedContents.universe;
+    await runCommand("yext", ["init", businessId, "-u", universe]);
+  } catch (error) {
+    console.error(
+      "Could not parse .yextrc file properly. Please make sure it is formatted correctly with a valid businessId and universe (valid options include sandbox or production)."
+    );
+    process.exit(1);
+  }
+};
+
+const runCommand = (command: string, args: string[]) => {
```

Comment:
Does the `yext init` output get written to the console? If not it should.

Reply:
User is able to see output of yext init command. Not error messages because of my own `console.error()` messages.